<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BodyJointEditorBase</class>
 <widget class="QWidget" name="BodyJointEditorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Bodyjoint name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>anchor x:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider0">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd0"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>anchor y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd1"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>anchor z:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd2"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="BodyJointEditorBaseLayout">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Degrees of Freedom</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="DofEditor" name="dofEditor0" native="true"/>
      </item>
      <item>
       <widget class="DofEditor" name="dofEditor1" native="true"/>
      </item>
      <item>
       <widget class="DofEditor" name="dofEditor2" native="true"/>
      </item>
      <item>
       <widget class="DofEditor" name="dofEditor3" native="true"/>
      </item>
      <item>
       <widget class="DofEditor" name="dofEditor4" native="true"/>
      </item>
      <item>
       <widget class="DofEditor" name="dofEditor5" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DofEditor</class>
   <extends>QWidget</extends>
   <header location="global">widgets/DofEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>slider0</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd0</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd1</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
